scripts/gen-s-parser.py

```diff
@@ -584,22 +582,14 @@
     ("br_on_non_data",       "makeBrOn(s, BrOnNonData)"),
     ("br_on_i31",            "makeBrOn(s, BrOnI31)"),
     ("br_on_non_i31",        "makeBrOn(s, BrOnNonI31)"),
-    ("rtt.canon",            "makeRttCanon(s)"),
-    ("rtt.sub",              "makeRttSub(s)"),
-    ("rtt.fresh_sub",        "makeRttFreshSub(s)"),
-    ("struct.new_with_rtt",  "makeStructNew(s, false)"),
-    ("struct.new_default_with_rtt", "makeStructNew(s, true)"),
     ("struct.new",           "makeStructNewStatic(s, false)"),
     ("struct.new_default",   "makeStructNewStatic(s, true)"),
     ("struct.get",           "makeStructGet(s)"),
     ("struct.get_s",         "makeStructGet(s, true)"),
     ("struct.get_u",         "makeStructGet(s, false)"),
     ("struct.set",           "makeStructSet(s)"),
-    ("array.new_with_rtt",   "makeArrayNew(s, false)"),
-    ("array.new_default_with_rtt", "makeArrayNew(s, true)"),
     ("array.new",            "makeArrayNewStatic(s, false)"),
     ("array.new_default",    "makeArrayNewStatic(s, true)"),
-    ("array.init",           "makeArrayInit(s)"),
```

Comment:
Hmm `array.init` was remved from parser but still exists in IR and C/JS-API. Maybe it was removed by accident?

Reply:
It remains in the IR because array.init_static still exists, but maybe the APIs could be cleaned up more.

Reply:
got it, thanks!